.github/templates/publish-release.yml

```diff
@@ -79,11 +142,14 @@ jobs:
         $nupkgName -match "(?sm)\d{1,2}\.\d{1,2}\.\d{1,2}(?:-[a-zA-Z]*\.\d{1,2})?"
         $version = $Matches[0]
         echo "::set-output name=version::$version"
-    - #@ template.replace(publishGithubRelease())
+    - #@ template.replace(prepareGithubRelease())
     - #@ template.replace(uploadDocsToS3())
     - #@ template.replace(uploadToNuget("Realm.Fody"))
     - #@ template.replace(uploadToNuget("Realm"))
     - #@ template.replace(uploadToNpm("io.realm.unity"))
+    - #@ template.replace(mergeReleasePR())
+    - #@ template.replace(publishGithubRelease())
```

Comment:
Is there a specific reason why we need to make a draft of the release first (line 145) and then publish after all the other actions (line 146-149) are done?

Reply:
No reason - I guess I thought it might be a good idea to have the release prepared if any of the following steps fails, but when I think about it, I'm not sure it adds that much value. I'll rework.